cmd/common-main.go

```diff
@@ -138,6 +138,9 @@ func minioConfigToConsoleFeatures() {
 	if value := env.Get(config.EnvBrowserRedirectURL, ""); value != "" {
 		os.Setenv("CONSOLE_BROWSER_REDIRECT_URL", value)
 	}
+	if value := env.Get(config.EnvConsoleDebugLogLevel, ""); value != "" {
+		os.Setenv("CONSOLE_DEBUG_LOGLEVEL", value)
+	}
```

Comment:
I added this, because it looks like all environment variables in MinIO should start with `MINIO_`.

Reply:
that is correct, ENVs must start with MINIO_